CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ### Improvements
 
 ### Changes
+- Adds a default for max expressions to be evaluated.
```

Comment:
We should say what that default is in the changelog, and make sure it's noted in the docstrings for the option as well.